mission_control/CMakeLists.txt

```diff
@@ -108,6 +108,4 @@ if(CATKIN_ENABLE_TESTING)
   add_rostest(test/test_jaus_ros_bridge_interface_mission_control.test)
   add_rostest(test/test_mission_control_publishes_heartbeat.test)
   add_rostest(test/test_mission_control_behaviors.test)
-  #catkin_add_gtest(behavior-object-test test/test_behavior_object.cpp)
-  #target_link_libraries(behavior-object-test ${catkin_LIBRARIES} ${PROJECT_NAME})
```

Comment:
@patapufate meta: we should add tests for reactive and introspectable action nodes though.